lib/components/Accordion/index.tsx

```diff
@@ -0,0 +1,155 @@
+};
+
+const AccordionVariant = {
+    bordered: "deriv-accordion__wrapper--bordered",
+    shadow: "deriv-accordion__wrapper--shadow",
+    underline: "deriv-accordion__wrapper--underline",
+} as const;
+
+const AccordionSection = memo(
+    ({
+        disableAnimation = false,
+        isActiveSection,
+        isCompact = false,
+        section,
+        sectionIndex,
+        setActiveIndex,
+        variant = "underline",
+    }: AccordionSectionProps) => {
+        const toggleSection = () => {
+            const nextIndex = isActiveSection ? null : sectionIndex;
+            setActiveIndex(nextIndex);
+        };
+
+        const [height, setHeight] = useState("");
+        const heightRef = useRef<HTMLDivElement>(null);
+
+        useEffect(() => {
+            const scrollHeight = heightRef.current?.scrollHeight;
+            setHeight(`${scrollHeight}px`);
+        }, [isActiveSection]);
+
+        return (
+            <div
+                className={clsx(
+                    "deriv-accordion__wrapper",
+                    AccordionVariant[variant],
+                    {
+                        "deriv-accordion__wrapper--compact": isCompact,
+                    },
+                )}
+            >
+                <div
+                    className={clsx("deriv-accordion__header", {
+                        "deriv-accordion__header--active": isActiveSection,
+                    })}
+                >
+                    <div>{section.title}</div>
+                    <div
+                        className="deriv-accordion__icon"
+                        onClick={toggleSection}
+                    >
+                        <img
+                            src={Chevron}
+                            className={clsx("deriv-accordion__icon", {
+                                "deriv-accordion__icon--active":
+                                    isActiveSection,
+                            })}
+                        />
+                    </div>
+                </div>
+                {isActiveSection && (
+                    <div
+                        className={clsx("deriv-accordion__content", {
+                            "deriv-accordion__content--active": isActiveSection,
+                            "deriv-accordion__content--compact": !isCompact,
+                            "deriv-accordion__content--no-animations":
+                                disableAnimation,
+                        })}
+                        ref={heightRef}
+                        style={{ maxHeight: isActiveSection ? height : "0px" }}
+                    >
+                        {section.content}
+                    </div>
+                )}
+            </div>
+        );
+    },
+);
+
+export const Accordion = ({
+    allowMultiple = false,
+    disableAnimation = false,
+    isCompact = false,
+    sections,
+    variant,
+}: AccordionProps) => {
+    const [activeIndexes, setActiveIndexes] = useState<number[]>([]);
```

Comment:
can we also have a prop for the seleted_indexes? 
just like tab component they should be able to control the selected item from their side as well. 
```suggestion
    const [activeIndexes, setActiveIndexes] = useState<number[]>(selectedIndexes || []);
    
        useEffect(() => {
        if (selectedIndexes) {
            setActiveIndexes(selectedIndexes);
        }
    }, [selectedIndexes]);
```